app/src/main/java/org/oppia/app/topic/play/TopicPlayFragment.kt

```diff
@@ -8,17 +8,39 @@ import android.view.ViewGroup
 import org.oppia.app.fragment.InjectableFragment
 import javax.inject.Inject
 
+internal const val KEY_CURRENT_EXPANDED_LIST_INDEX = "CURRENT_EXPANDED_LIST_INDEX"
```

Comment:
Prefer this being private if possible to minimize visibility.

Reply:
done.